kernel/build/config-amd64

```diff
@@ -2392,7 +2392,7 @@ CONFIG_NET_VENDOR_ATHEROS=y
 # CONFIG_ATL1 is not set
 # CONFIG_ATL1E is not set
 # CONFIG_ATL1C is not set
-# CONFIG_ALX is not set
```

Comment:
if we build as modules, we should include them as the list of files we copy by default to `initramfs`